libapr.yaml

```diff
@@ -0,0 +1,78 @@
+package:
+  name: libapr
+  version: 1.7.4
+  epoch: 0
+  description: "Apache Portable Runtime Library (APR)"
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - wolfi-base
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - elfutils-dev
+      - linux-headers
+      - zstd-dev
+      - zlib-dev
+      - automake
+      - autoconf
+      - python3
+      - libtool
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/apache/apr
+      tag: ${{package.version}}
+      expected-commit: 22867ddac49b7dcf356556577461231acdff6445
+
+  - uses: patch
+    with:
+      patches: apr-1.7.2-autoconf-2.72.patch
+
+  - runs: |
+      ./buildconf
+
+  - if: ${{build.arch}} == 'x86_64'
```

Comment:
If we ever add a third arch, this will fail since configure hasn't run. Can we add another step that fails if neither of the previous steps ran, to get a clearer error? There's a precedent for go-stage0 I think.

Reply:
removed we don't need this anymore